Spotlight update for August 2024 - images, please merge to amada-spotlight-aug-2024

amada-ech opened the pull request:
## What are you changing in this pull request and why?
<!---
Describe your changes and why you're making them. If related to an open 
issue or a pull request on dbt Core, then link to them here! 

To learn more about the writing conventions used in the dbt Labs docs, see the [Content style guide](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/content-style-guide.md).
-->

## Checklist
<!--
Uncomment when publishing docs for a prerelease version of dbt:
- [ ] Add versioning components, as described in [Versioning Docs](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/single-sourcing-content.md#versioning-entire-pages)
- [ ] Add a note to the prerelease version [Migration Guide](https://github.com/dbt-labs/docs.getdbt.com/tree/current/website/docs/docs/dbt-versions/core-upgrade)
-->
- [ ] Review the [Content style guide](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/content-style-guide.md) so my content adheres to these guidelines.
- [ ] For [docs versioning](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/single-sourcing-content.md#about-versioning), review how to [version a whole page](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/single-sourcing-content.md#adding-a-new-version) and [version a block of content](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/single-sourcing-content.md#versioning-blocks-of-content).
- [ ] Add a checklist item for anything that needs to happen before this PR is merged, such as "needs technical review" or "change base branch."

Adding or removing pages (delete if not applicable):
- [ ] Add/remove page in `website/sidebars.js`
- [ ] Provide a unique filename for new pages
- [ ] Add an entry for deleted pages in `website/vercel.json`
- [ ] Run link testing locally with `npm run build` to update the links that point to deleted pages

amada-ech committed:
Spotlight update for August 2024 - images

amada-ech committed:
Create mikko-sulonen.md

Mikko Sulonen's card! 

Please note It should pull in his headshot, mikko-sulonen, which I added in here https://github.com/dbt-labs/docs.getdbt.com/pull/5866

amada-ech committed:
Create meagan-palmer.md

Card for Meagan Palmer!

amada-ech committed:
Update index.js

edited line 4 to "const currentSpotlightDate = new Date('2024-07-31')"

so that Maegan, Radovan and Mikko show up ABOVE Previously on the Spotlight

The dates I chose for them are July 28, 29 and 30. And the spotlight instructions say to put the const currentSpotlightDate  equal to 1 day after the new spotlight member submissions

if it works, Maegan, Radovan and Mikko should be at the top (not in that exact order) and then it should say Previously on the Spotlight below that, and have all the previous folks

amada-ech committed:
Update docusaurus.config.js

updated like 84 from tyler-rouze to meagan-palmer so that she appears as the featured member on Dev Hub!

mirnawong1 committed:
Merge branch 'current' into amada-spotlight-aug-2024-1

mirnawong1 committed:
Update meagan-palmer.md

mirnawong1 committed:
Update mikko-sulonen.md

mirnawong1 commented on website/docs/community/spotlight/meagan-palmer.md:
```diff
@@ -0,0 +1,41 @@
+title: Meagan Palmer
+description: |
+  I first started using dbt in 2016 or 2017 (I can't remember exactly). Since then, I have moved into data and analytics consulting and have dipped in and out of the dbt Community. 
+  Late last year, I started leading dbt Cloud training courses and spending more time in the <a href="https://www.getdbt.com/community/join-the-community/“ rel="noopener noreferrer" target="_blank">dbt Slack</a>. 
```
i think this is what could be causing the error - should it be:  `<a target="_blank" rel="noopener noreferrer" href="https://www.getdbt.com/community/join-the-community/">dbt Slack</a>` instead?

john-rock replied:
I think you're correct @mirnawong1. It looks like that fancy quote after the href is causing a parsing error in the build.
![Spotlight_update_for_August_2024_-_images__please_merge_to_amada-spotlight-aug-2024](https://github.com/user-attachments/assets/be1e7460-0442-4b82-9517-e98a4e6720c4)

mirnawong1 committed:
Merge branch 'current' into amada-spotlight-aug-2024-1

mirnawong1 commented on website/docs/community/spotlight/meagan-palmer.md:
```diff
@@ -0,0 +1,41 @@
+title: Meagan Palmer
+description: |
+  I first started using dbt in 2016 or 2017 (I can't remember exactly). Since then, I have moved into data and analytics consulting and have dipped in and out of the dbt Community. 
+  Late last year, I started leading dbt Cloud training courses and spending more time in the <a href="https://www.getdbt.com/community/join-the-community/“ rel="noopener noreferrer" target="_blank">dbt Slack</a>. 
```
i think this is what could be causing the error - should it be this instead?
```suggestion
  Late last year, I started leading dbt Cloud training courses and spending more time in the <a target="_blank" rel="noopener noreferrer" href="https://www.getdbt.com/community/join-the-community/">dbt Slack</a>. 
```

mirnawong1 commented on website/docs/community/spotlight/meagan-palmer.md:
```diff
@@ -0,0 +1,41 @@
+  I first started using dbt in 2016 or 2017 (I can't remember exactly). Since then, I have moved into data and analytics consulting and have dipped in and out of the dbt Community. 
+  Late last year, I started leading dbt Cloud training courses and spending more time in the <a href="https://www.getdbt.com/community/join-the-community/“ rel="noopener noreferrer" target="_blank">dbt Slack</a>. 
+  In consulting, I get to use a range of stacks. I've used dbt with Redshift, Snowflake, and Databricks in production settings with a range of loaders & reporting tools, and I've been enjoying using DuckDB for some home experimentation. 
+  To share some of the experiences, I regularly post to LinkedIn and have recently started <a href="https://www.linkedin.com/newsletters/analytics-engineering-today-7210968984693690370/“ rel="noopener noreferrer" target="_blank">Analytics Engineering Today</a>, a twice monthly newsletter about dbt in practice.
```
```suggestion
  To share some of the experiences, I regularly post to LinkedIn and have recently started <a target="_blank" rel="noopener noreferrer" href="https://www.linkedin.com/newsletters/analytics-engineering-today-7210968984693690370/“>Analytics Engineering Today</a>, a twice monthly newsletter about dbt in practice.
```

john-rock reviewed (commented):
Hey @amada-ech I believe we just need to replace the `“` with `"` at the end of the href values to get this deploy to build successfully.

Everything else looks good but I'll give it another look once those adjustments are made.

john-rock committed:
update files

amada-ech committed:
Update index.js

Updated line 14 from
const currentSpotlightDate = new Date('2024-07-31') 
to
const currentSpotlightDate = new Date('2024-07-26') 

Goal is for Maegan, Radovan and Mikko to show up above "Previously on the Spotlight." They are currently below

amada-ech committed:
Update meagan-palmer.md

edited the linkedin link 
from
www.linkedin.com/in/meaganpalmer
to 
https://www.linkedin.com/in/meaganpalmer/

It was taking me to Page not Found for some reason!

Also changed the question "What community member...." to "Which community member....".

amada-ech committed:
Update mikko-sulonen.md

Changed "## What dbt community leader do you identify with? How are you looking to grow your leadership in the dbt community?"

to 
## Which dbt community leader do you identify with? How are you looking to grow your leadership in the dbt community?

amada-ech committed:
Update radovan-bacovic.md

updated link for Sean McIntyre hyperlink to https://www.linkedin.com/in/boxysean/

mirnawong1 committed:
Merge branch 'current' into amada-spotlight-aug-2024-1

john-rock reviewed (approved):
lgtm after the latest edits!

john-rock committed:
Merge branch 'current' into amada-spotlight-aug-2024-1

john-rock commented:
Merged for @amada-ech since she did not have the permissions to do so.
